trace/google/cloud/trace_v2/proto/tracing_pb2.py

```diff
@@ -0,0 +1,274 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: google/devtools/cloudtrace_v2/proto/tracing.proto
+
+import sys
+_b=sys.version_info[0]<3 and (lambda x:x) or (lambda x:x.encode('latin1'))
+from google.protobuf import descriptor as _descriptor
+from google.protobuf import message as _message
+from google.protobuf import reflection as _reflection
+from google.protobuf import symbol_database as _symbol_database
+from google.protobuf import descriptor_pb2
+# @@protoc_insertion_point(imports)
+
+_sym_db = _symbol_database.Default()
+
+
+from google.api import annotations_pb2 as google_dot_api_dot_annotations__pb2
+from google.devtools.cloudtrace_v2.proto import trace_pb2 as google_dot_devtools_dot_cloudtrace__v2_dot_proto_dot_trace__pb2
```

Comment:
Importing from `google.devtools` is causing the tests failure, can we rename it?